dkc/core/models/authorized_upload.py

```diff
@@ -0,0 +1,34 @@
+from django.contrib.auth.models import User
+from django.core import signing
+from django.db import models
+from django_extensions.db.models import CreationDateTimeField
+
+from .folder import Folder
+
+
+class AuthorizedUpload(models.Model):
+    folder = models.ForeignKey(
+        Folder, on_delete=models.CASCADE, related_name='authorized_uploads', editable=False
+    )
+    creator = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name='authorized_uploads', editable=False
+    )
+    created = CreationDateTimeField()
+    expires = models.DateTimeField(editable=False)
```

Comment:
Could `expires` have a default value function, with the logic (currently in the view layer):
```
return timezone.now() + timedelta(days=settings.DKC_AUTHORIZED_UPLOAD_EXPIRATION_DAYS)
```

Reply:
Alternatively, it's not really necessary to store `expires` as a column at all, since we know `created` and `DKC_AUTHORIZED_UPLOAD_EXPIRATION_DAYS`. It could just be a computed `@property`.